extensions/wikia/PortableInfobox/services/PortableInfoboxMobileRenderService.class.php

```diff
@@ -127,7 +127,7 @@ protected function renderTitle( $data ) {
 	}
 
 	protected function renderHeader( $data ) {
-		return $this->render( 'header', $data );
+        return $this->render( 'header-mobile', $data );
```

Comment:
@bkoval does it work at all? https://github.com/Wikia/app/pull/15821/commits/177608003c5980217ecd84f753129366c19ef4a5#diff-1b81450a159689d93eae69120766ee31R15 has `mobile-header` as a key ;)